NineChronicles.Headless.Executable.Tests/Commands/ChainCommandTest.cs

```diff
@@ -27,6 +26,10 @@ namespace NineChronicles.Headless.Executable.Tests.Commands
 {
     public class ChainCommandTest : IDisposable
     {
+        /// <summary>
+        /// `StoreType.Memory` will not be tested.
+        /// Because the purpose of ChainCommandTest is to store and read blockchain data.
+        /// </summary>
```

Comment:
In my thought, it shouldn't be an XML document here. The document will describe `ChainCommandTest._console` variable. How about describing it above the `ChainCommandTest` as normal documents (`//` not `///`)?

Reply:
I didn't know the difference of `///` and `//` type comments.
I'll change them to normal comment.